### DISCUSSION TITLE
How to use the  Milvus WebUI?

### DISCUSSION THREAD

**Discussion by tangkunyin** [milvus-io/milvus#38806] [Q&A and General discussion] [answered]
I know the `webui` is a built-in management tool for both operations and development from the [official document](https://milvus.io/docs/milvus-webui.md), but I have tried http://localhost:9091/webui/ and got 404...I searched google and project issues, but no useful information found!

**I have a question, is the feature supported from v2.5.x? If not, how to use it?**

My services under docker container were all running well, the version of milvus standalone is v2.4.17

**Comment by yhmo** [comment, marked as answer]
v2.4.x doesn't support webui.
You can deploy a standalone v2.5.0, access the webui by **http://<MILVUS_PROXY_IP>:9091/webui/**.
It is mainly for operations team and developer team.
![Screenshot from 2024-12-27 14-40-32](https://github.com/user-attachments/assets/e4104c3a-7ffc-4f1b-9a16-2973804a5e48)
